src/app.js

```diff
@@ -71,7 +71,6 @@ app.use('/user', usersRouter)
 
 // Error handlers
 // Use HB before all other error handlers.
-app.use(Honeybadger.errorHandler)
```

Comment:
part 2 - this removes the handler that alerted HB automatically at all times - see https://www.honeybadger.io/blog/honeybadger-for-node-js-is-now-awesome/